### ISSUE TITLE
Buggy behavior when remapping 'v'

### ISSUE THREAD

**Issue by jgregoire** [ggandor/leap-spooky.nvim#14]
Expected behavior: yarw copies a remote word

Actual behavior: yarw copies everything between cursor and leap target and autopastes it, despite setting `paste_on_remote_yank = false`

I eventually figured out that commenting out my remapping of 'p' to 'v' fixes the issue. Using legendary.nvim:
```
keymaps = {
    { 'v', 'p', mode = {  'n' }, opts = { noremap = true } },
},
```
causes the weird behavior. Took me a while to isolate this one!

Legendary keybinds are set before calling `require('leap-spooky').setup()`

Using latest version of leap and leap-spooky, nvim 0.8.3 and 0.9.0 nightly.

**Comment by ggandor** [comment]
Yeah, the problem is that we use `normal` instead of `normal!` here:

https://github.com/ggandor/leap-spooky.nvim/blob/3e940464b1728b22052dd565accc949c0b02b025/lua/leap-spooky.lua#L47
